website_supplier/views/res_partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="website_supplier_form_view">
+            <field name="name">Website Supplier</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="website_partner.view_partners_form_website"/>
+            <field name="arch" type="xml">
+                <field name="website_published" position="after">
+                    <field name="website_supplier_published" 
+                        attrs="{'invisible': [('supplier','=', False)]}"/>
```

Comment:
Almost all Objects that are optional to be published on the website use the `widget="website_button"` widget, can you add this attr to keep the UI consistent?